### PR TITLE
chore: Update deployment scripts

### PR DESCRIPTION
**Pull request by shyakadavis**
Saw you were still using node 18.x, and left it as such, but could update if needed.

TODO:

- [ ] Disable Cloudflare's GitHub integration on the repo
- [ ] Add variable secrets to this repo (`CF_ACCOUNT_ID` & `CF_API_TOKEN`)
- [ ] Confirm that the project name is `layerchart` on Cloudflare

### CODE REVIEW

**Comment by shyakadavis** [comment]
Huh, still seeing Vercel trying to deploy. Was I wrong in assuming that you're using CF now, Sean?

**Comment by techniq** [comment]
Thanks @shyakadavis ❤️.  I re-enabled Vercel after I learned that CF didn't provide PR previews, and utilize the [adapter-auto](https://github.com/techniq/layerchart/blob/main/packages/layerchart/svelte.config.js#L1) ATM to deploy to both (CF is the primary and where layerchart.com is hosted).  Since I had stopped paying for a Vercel Pro account, this means the PR previews are really only for me, which isn't optimal but still useful for me to review PRs.

So in short, yes I am still using Vercel mostly as a stop gap solution for PR previews, but likely turning it off after this.  It's been helpful once or twice to troubleshoot a CF-specific failing build, but likely will disable it to cut down on the PR comment noise.

Thanks so much for the PR.  I plan to finish the setup this afternoon when I get back home.

**Review by techniq — APPROVED** [review]
Thanks @shyakadavis!

**Comment by techniq** [comment]
@shyakadavis LayerChart and Svelte UX deployments are all migrated over 🎉

- https://github.com/techniq/layerchart/pull/224#issuecomment-2267964196
- https://github.com/techniq/svelte-ux/pull/436#issuecomment-2267952133

I switched from `adapter-auto` to `adapter-cloudflare` as I didn't see a straightforward way to support `adapter-auto` since the build is happening in Github Actions instead of Cloudflare infra.  I also removed the Vercel integration since it would need setup similarly without using adapter-auto, and not worth it now 😁 (PR previews for everyone!)

Thanks again!
